kubernetes/k8_workers.tf

```diff
@@ -101,4 +101,35 @@ module "kubernetes_workers" {
       }
     }
   }
+  vault_agent = {
```

Comment:
The commit of submodule `terraform-libvirt-kubernetes-node` needs to be changed so this var is available.